packages/@expo/config-plugins/src/android/__tests__/BuildProperties-test.ts

```diff
@@ -148,4 +148,15 @@ describe(updateAndroidBuildProperty, () => {
       })
     ).toEqual([{ type: 'property', key: 'foo', value: 'foo' }]);
   });
+
+  it('should merge properties when `value` is an array', () => {
+    const gradleProperties: PropertiesItem[] = [
+      { type: 'property', key: 'foo', value: 'foo' },
+      { type: 'property', key: 'somearray', value: `["name1"]` },
+    ];
+    expect(updateAndroidBuildProperty(gradleProperties, 'somearray', `["name2"]`)).toEqual([
+      { type: 'property', key: 'foo', value: 'foo' },
+      { type: 'property', key: 'somearray', value: `["name1","name2"]` },
```

Comment:
This seems like a breaking change as any plugin will now stack values when prebuild is run multiple times without the clean flag. This could lead to unexpected bugs. Probably better if there's a solution where a plugin can read the gradle properties and "ensure" a given value is included. Perhaps using a custom mod with `withGradleProperties` is a better approach for library-based plugins.

Reply:
Any change that would be made by withGradleProperties would get wiped out by any later call to any other thing that depends on withGradleProperties, so I'm not sure I follow 🤔 .
I'll try to update my code so that it is "re-run" safe, because for now I don't really see any other solution!